src/Components/Molecules/UserHeaderMolecule/index.tsx

```diff
@@ -0,0 +1,26 @@
+import {useState} from "react";
+import {useEffect} from "react";
+import HeaderAtom from "../../Atoms/HeaderAtom";
+
+function UserHeader() {
+
+    const [customerName, setCustomerName] = useState('Liz')
```

Comment:
I know you used 'Liz' just to test this, so make sure to replace this with '' now